Content-Organization.md

```diff
@@ -23,6 +23,7 @@ The Egeria content is organized into the following modules:
     * **[design-model](open-metadata-implementation/access-services/design-model)** - manage content from design models.
     * **[dev-ops](open-metadata-implementation/access-services/dev-ops)** - manage metadata for a devOps pipeline.
     * **[digital-architecture](open-metadata-implementation/access-services/digital-architecture)** - support the definition of data standards and models.
+    * **[digital-service](open-metadata-implementation/access-services/digital-service)** - manage metadata for an Egeria Digital Service.
```

Comment:
Remove `Egeria` - digital service belongs to organization not Egeria.

Reply:
resolved